includes/class-amp-story-post-type.php

```diff
@@ -936,6 +936,31 @@ public static function enqueue_block_editor_scripts() {
 			);
 		}
 
+		/**
+		 * Filter list of allowed video mime types.
+		 *
+		 * This can be used to add additionally supported formats, for example by plugins
+		 * that do video transcoding.
+		 *
+		 * @since 1.2
+		 *
+		 * @param array Supported video mime types.
+		 */
+		$allowed_video_mime_types = apply_filters( 'amp_supported_video_types', [ 'video/mp4' ] );
```

Comment:
```suggestion
		$allowed_video_mime_types = apply_filters( 'amp_story_supported_video_types', [ 'video/mp4' ] );
```

Let's make it clear that this is for stories only.